lib/ome/common/module.cpp

```diff
@@ -76,9 +76,9 @@ namespace
     // Global paths (not specific to any component)
     static RegisterModule bin
       ("bin",
-       "BIOFORMATS_BINDIR",
+       "OME_FILES_BINDIR",
```

Comment:
This depends entirely on the naming conventions we're adopting.  This is intended to be the program directory for all ome-files programs.  This was done on the assumption that ome-files was going to be used as the top-level name for all the collated components.  If that's not the case then we need to decide on a proper name for that (which kind of interferes with ome-files being the "marketing name" to my mind), and we also need a concrete definition of what ome-files actually is, because it's still far too vague for me to do anything concrete with if this is still all not properly decided upon.

Note that I was going to follow up the module changes with a further one to allow the registration of the same name by different components, so that each one can have its own bin dir etc.  That would mean that the ome-files component would register its own as well, and this one would be the one-common one.  But... we still need a name for the entire set...